src/renderer/components/dialogs/CreateChat.tsx

```diff
@@ -194,7 +194,7 @@ export function useContactSearch(
   }
 
   const onSearchChange = (e: React.ChangeEvent<HTMLInputElement>) =>
-    updateSearch(e.target.value)
+    updateSearch(e.target.value.trim())
```

Comment:
I would instead do this before sending to core and allow users to type spaces there

Reply:
the function to create a new contact is in Line 412